crm_phonecall/README.rst

```diff
@@ -0,0 +1,111 @@
+===============
+CRM Phone Calls
+===============
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fcrm-lightgray.png?logo=github
+    :target: https://github.com/OCA/crm/tree/15.0/crm_phonecall
+    :alt: OCA/crm
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/crm-15-0/crm-15-0-crm_phonecall
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/webui/builds.html?repo=OCA/crm&target_branch=15.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to manage phone calls in order to analyze them.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to *CRM > Phone Calls > Logged Calls > Create*.
+#. If your user has *Show Scheduled Calls Menu* permission, you will see
+   scheduled calls menu too.
+#. In any moment you can schedule another call, schedule a meeting or convert
+   call contact to opportunity.
+#. Calls can be categorized and you can manage categories in *CRM >
+   Configuration > Pipeline > Phone Calls > Categories*.
```

Comment:
*CRM >
   Configuration > Phone Calls > Categories*

Reply:
Change done. Thanks for the suggestions.